java/src/jmri/jmrix/AbstractMRTrafficController.java

```diff
@@ -53,7 +53,7 @@ public AbstractMRTrafficController() {
         Runtime.getRuntime().addShutdownHook(shutdownHook);
     }
 
-    private boolean synchronizeRx = true;
+    protected boolean synchronizeRx = true;
     
```

Comment:
Please make this private again and add a Getter method instead.

Reply:
done